dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java

```diff
@@ -134,14 +137,16 @@
     @Enumerated(EnumType.STRING)
     @ApiModelProperty(value = "This is a specific docker provider like quay.io or dockerhub or n/a?, required: GA4GH", required = true)
     private Registry registry;
-    @Column
-    @ApiModelProperty(value = "This is a generated full docker path including registry and namespace, used for docker pull commands")
-    private String path;
 
     @Column
     @ApiModelProperty("Implementation specific timestamp for last built")
     private Date lastBuild;
 
+    @Column
+    @JsonProperty("custom_docker_registry_path")
+    @ApiModelProperty(value = "Only used for docker registries that allow for custom paths")
```

Comment:
Make an issue to add this as an actual database constraint

Reply:
This column will actually be removed in my next PR since we will store the docker registry path in the registry column instead of the enum.